packages/expo-auth-session/src/ManagedSessionUrlProvider.ts

```diff
@@ -86,7 +86,7 @@ export class ManagedSessionUrlProvider implements SessionUrlProvider {
   }
 
   private static getHostAddress(): { hostUri: string; parameters: string | undefined } {
-    let hostUri: string = Constants.manifest.hostUri;
+    let hostUri: string = Constants.manifest?.hostUri;
```

Comment:
There will also be a problem with `Constants.linkingUri`. We should probably just log a warning in dev mode rather than throwing an error.